### DISCUSSION TITLE
How to set run timeout?

### DISCUSSION THREAD

**Discussion by AsimAM** [sorry-cypress/sorry-cypress#434] [Q&A]
It defaults to 3 minutes, from doing some digging I can see that we should be able to set `inactivityTimeoutSeconds` in `cypress.json` but it doesn't seem to work

**Comment by agoldis** [comment]
It's in the UI - edit project > set timeout > save

**Comment by AsimAM** [comment]
it should work through the cypress configuration though no? It's a bit of a pain to have to use the UI every time a project is created. In my own team we have a setup which clears the data every month for example, so when a new project is automatically created someone will manually have to go in update the timeout through the UI which isn't really maintainable in the long run. @agoldis

**Comment by agoldis** [comment]
you can also set the default timeout via director's runtime environment variable `INACTIVITY_TIMEOUT_SECONDS` https://github.com/sorry-cypress/sorry-cypress/blob/39badda95279b9bb844b5cae75fcf6d663518df9/packages/director/src/config.ts#L26:L26
